developers/command-api.rst

```diff
@@ -0,0 +1,169 @@
+The Command API
+==========================
+
+What better way to expand your plugins using commands? This page will show how
```

Comment:
Try to take a more professional tone here. Also, avoid using sentences of the form "this X will show you how to Y".

Reply:
I took the first tutorial as an example. https://github.com/VelocityPowered/documentation/blame/master/developers/creating-your-first-plugin.rst#L4

Reply:
I was also particularly rushed in writing that page. Try to use the user-facing documentation instead - that's what i want.